packages/core/src/transactions/utils/simplePath.ts

```diff
@@ -0,0 +1,192 @@
+import { AbiItem, isAddress } from 'web3-utils'
+
+import TransactionPath from '../TransactionPath'
+import TransactionRequest from '../TransactionRequest'
+import { createDirectTransactionForApp } from './transactions'
+import App from '../../wrappers/App'
+import Organization from '../../wrappers/Organization'
+import { Permission } from 'plumbery-connector-thegraph/src/graphql/types'
+
+export interface AragonArtifactFunction {
+  roles: string[]
+  sig: string
+  /**
+   * This field might not be able if the contract does not use
+   * conventional solidity syntax and Aragon naming standards
+   * null if there in no notice
+   */
+  notice: string | null
+  /**
+   * The function's ABI element is included for convenience of the client
+   * null if ABI is not found for this signature
+   */
+  abi: AbiItem | null
+}
+
+export const ANY_ENTITY = '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF'
+
+export function addressesEqual(first: string, second: string): boolean {
+  first = first && first.toLowerCase()
+  second = second && second.toLowerCase()
+  return first === second
+}
+
+// "Safer" version of [].includes() for addresses
+export function includesAddress(arr: any[], address: string): boolean {
```

Comment:
I think we could use `string[]` here?

```suggestion
export function includesAddress(arr: string[], address: string): boolean {
```